internal/api/rbac/policy_rules_test.go

```diff
@@ -10,61 +10,22 @@ import (
 )
 
 func TestNormalizePolicyRules(t *testing.T) {
-
-	t.Run("wildcard group not allowed", func(t *testing.T) {
-		_, err := NormalizePolicyRules([]rbacv1.PolicyRule{
-			{
-				APIGroups: []string{"*"},
-				Resources: []string{"pods"},
-				Verbs:     []string{"get"},
-			},
-		})
-		require.ErrorContains(t, err, "wildcard APIGroup is not allowed")
-	})
-
-	t.Run("wildcard resource not allowed", func(t *testing.T) {
+	t.Run("invalid resource type", func(t *testing.T) {
 		_, err := NormalizePolicyRules([]rbacv1.PolicyRule{
 			{
 				APIGroups: []string{""},
-				Resources: []string{"*"},
-				Verbs:     []string{"get"},
-			},
-		})
-		require.ErrorContains(t, err, "wildcard Resource is not allowed")
-	})
-
-	t.Run("multiple groups expand", func(t *testing.T) {
-		rules, err := NormalizePolicyRules([]rbacv1.PolicyRule{
-			{ // Never mind that this doesn't make sense
-				APIGroups: []string{"", rbacv1.GroupName},
-				Resources: []string{"pods"},
+				Resources: []string{"fake-resource"},
 				Verbs:     []string{"get"},
 			},
 		})
-		require.NoError(t, err)
-		require.Equal(
-			t,
-			[]rbacv1.PolicyRule{
-				{
-					APIGroups: []string{""},
-					Resources: []string{"pods"},
-					Verbs:     []string{"get"},
-				},
-				{
-					APIGroups: []string{rbacv1.GroupName},
-					Resources: []string{"pods"},
-					Verbs:     []string{"get"},
-				},
-			},
-			rules,
-		)
+		require.ErrorContains(t, err, "unknown resource type")
 	})
 
 	t.Run("multiple resources expand", func(t *testing.T) {
 		rules, err := NormalizePolicyRules([]rbacv1.PolicyRule{
 			{
 				APIGroups: []string{""},
-				Resources: []string{"pods", "services"},
```

Comment:
Can't use pods as a resource type in these tests anymore since it isn't recognized as valid anymore. There's no reason for pod permissions to be added to roles in a Kargo project.